carbon-route-converter.html

```diff
@@ -20,18 +20,30 @@
 an app, please refer to [carbon-location](https://github.com/PolymerElements/carbon-route/blob/master/carbon-location.html)
 and [carbon-route](https://github.com/PolymerElements/carbon-route/blob/master/carbon-route.html).
 
-An example of a route element that describes
-`https://elements.polymer-project.org/elements/carbon-route-converter?foo=bar&baz=qux`:
+An example of a route object that describes
+`https://elements.polymer-project.org/elements/carbon-route-converter?foo=bar&baz=qux`
+and should be passed to other `carbon-route` elements:
 
     {
       prefix: '',
       path: '/elements/carbon-route-converter',
-      queryParams: {
+      __queryParams: {
         foo: 'bar',
         baz: 'qux'
       }
     }
 
+The private `__queryParams` property in the `route` object should be used only
```

Comment:
V good points. Is your name actually Peter Word Smith? I prefer your fancy letters.